egs/ami/s5/run_sdm.sh

```diff
@@ -10,17 +10,16 @@ mic=sdm$micid
 stage=0
 . utils/parse_options.sh
 
-# Set bash to 'debug' mode, it will exit on :
-# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
-set -e
-set -u
-set -o pipefail
-set -x
+# Set bash to 'debug' mode, it prints the commands (option '-x') and exits on : 
+# -e 'error', -u 'undefined variable', -o pipefail 'error in pipeline',
+set -euxo pipefail
 
 # Path where AMI gets downloaded (or where locally available):
-[ ! -r conf/ami_dir ] && echo "Please, run 'run_prepare_shared.sh' first!" && exit 1
-AMI_DIR=$(cat conf/ami_dir)
+AMI_DIR=$PWD/wav_db # Default,
+#AMI_DIR=/mnt/scratch05/iveselyk/KALDI_AMI_WAV # BUT,
+#AMI_DIR=/export/ws15-ffs-data/corpora/ami # JSALT2015 workshop, cluster AWS-EC2,
 
+[ ! -r data/local/lm/final_lm ] && echo "Please, run 'run_prepare_shared.sh' first!" && exit 1
```

Comment:
This comment corresponds to stage 0:
The common scenario on any cluster is that the data would have been downloaded once and reused many times. So in most cases there would be no <code>data/local/downloads/wget_sdm.sh</code> even though there is data in the AMI_DIR. It is better to check the AMI_DIR has the required data, before starting the download. I think this check can be best performed in the <code>local/ami_download.sh</code> script. (See my comment there)